packages/ember-glimmer/lib/components/link-to.ts

```diff
@@ -534,10 +518,14 @@ const LinkComponent = EmberComponent.extend({
   */
   disabled: computed({
     get(_key: string): boolean {
+      // always returns false for `get` because (due to the `set` just below)
+      // the cached return value from the set will prevent this getter from _ever_
+      // being called after a set has occured
       return false;
     },
+
     set(_key: string, value: any): boolean {
```

Comment:
Unrelated to the changes here, but the typing seems wrong. Shouldn't it be `set(...): string|false`?

Reply:
@simonihmig - Yes, this should be typed as a `string | null` I believe.